pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -521,9 +551,28 @@ func (r *SubnetPortReconciler) Start(mgr ctrl.Manager) error {
 	if err != nil {
 		return err
 	}
+	go r.pollVIFs(make(chan bool))
 	return nil
 }
 
+var ticker = time.NewTicker(10 * time.Minute)
```

Comment:
Shall we define ticker in pollVIFs function as it is not used by other function?